articles/search/search-filters.md

```diff
@@ -96,13 +96,13 @@ The following examples illustrate several usage patterns for filter scenarios. F
 + Standalone **$filter**, without a query string, useful when the filter expression is able to fully qualify documents of interest. Without a query string, there is no lexical or linguistic analysis, no scoring, and no ranking. Notice the search string is just an asterisk, which means "match all documents".
 
    ```
-   search=*&$filter=(baseRate ge 60 and baseRate lt 300) and accommodation eq 'Hotel' and city eq 'Nogales'
+   search=*&$filter=Rooms/any(room: room/BaseRate ge 60 and room/BaseRate lt 300) and Address/City eq 'Honolulu'
    ```
 
 + Combination of query string and **$filter**, where the filter creates the subset, and the query string provides the term inputs for full text search over the filtered subset. Using a filter with a query string is the most common usage pattern.
 
    ```
-   search=hotels ocean$filter=(baseRate ge 60 and baseRate lt 300) and city eq 'Los Angeles'
+   search=ooms/any(room: room/BaseRate ge 60 and room/BaseRate lt 300) and Address/City eq 'Seattle'
```

Comment:
Missing an R in Rooms, but it's also not working -- I get all 50 hotels back, not just the ones in Seattle.  I realize now its because the $filter got removed.  If you put it back (search=*$filter=Rooms/etc) plus add a search term like "walking distance theaters" then you get 3 hotels, with search scores, with the hotel with that phrase at the top.

This is the string that works for me:

search=walking distance theaters&$filter=Rooms/any(room: room/BaseRate ge 60 and room/BaseRate lt 300) and Address/City eq 'Seattle'&$count=true

Reply:
Fixed these! Yes it was copy paste, I realized that sometimes when we miss a '$' and some missing parameter, it still returns the results and does not throw any error. So these syntax are important.